draft-ietf-quic-transport.md

```diff
@@ -4400,9 +4448,9 @@ STREAM frames ensures that loss recovery, congestion control, and flow control
 operate effectively.
 
 CRYPTO frames SHOULD be prioritized over other streams prior to the completion
-of the cryptographic handshake.  This includes the retransmission of the second
-flight of client handshake messages, that is, the TLS Finished and any client
-authentication messages.
+of the cryptographic handshake.  In {{QUIC-TLS}} this includes the retransmission
+of the second flight of client handshake messages, that is the TLS Finished and
+any client authentication messages.
```

Comment:
Addition of QUIC-TLS reference seems unnecessary, and the removal of the comma is incorrect.

Reply:
this sentence is very TLS specfic and bundles the two doc more then needed if left as a general statement. 

Readded the comma...